docs/code_search/reference/queries.mdx

```diff
@@ -11,32 +11,52 @@ This page describes search pattern syntax and keywords available for code search
 
 This section documents the available search pattern syntax and interpretation in Sourcegraph. A search pattern is _required_ to match file content. A search pattern is _optional_ and may be omitted when searching for [commits](#keywords-diff-and-commit-searches-only), [filenames](#filename-search), or [repository names](#repository-name-search).
 
-### Standard search (default)
+### Keyword search (default)
 
-Standard search matches literal patterns exactly, including puncutation like quotes. Specify regular expressions inside `/./code_search/reference`.
+<Callout type="note" title="Experimental">New in version 5.3: Keyword search is in the experimental stage and enabled by default. Site administrators can disable keyword search globally by setting `experimentalFeatures.keywordSearch: false` in site settings. Users can override the setting in their user settings.</Callout>
```

Comment:
I believe we decided on calling this 'beta' instead of 'experimental'. Looking at our [docs for what experimental features mean](https://docs.sourcegraph.com/admin/beta_and_experimental_features#experimental-features), it doesn't seem like a good fit -- experimental features have limited support, have to be explicitly enabled, etc.